backend_modules/null/base/variables.tf

```
@@ -65,6 +65,6 @@ variable "provider_settings" {
 
 variable "images" {
   description = "list of images to be uploaded to the libvirt host, leave default for all"
-  default     = ["centos6o", "centos7o", "centos8o", "opensuse152o", "opensuse153o", "opensuse154o", "sles11sp4", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o", "amazonlinux2o", "almalinux8o", "rocky8o"]
+  default     = ["centos7o", "centos8o", "opensuse152o", "opensuse153o", "opensuse154o", "sles11sp4", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1804o", "ubuntu2004o", "amazonlinux2o", "almalinux8o", "rocky8o"]
```

```suggestion
  default     = ["centos7o", "centos8o", "opensuse153o", "opensuse154o", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1804o", "ubuntu2004o", "amazonlinux2o", "almalinux8o", "rocky8o"]
```

SLES15 older than SP3 is also out of generic support, but I guess since LTSS is there we might keep it here.